application/web-app/react-client/src/App.js

```diff
@@ -1,93 +0,0 @@
-/*
```

Comment:
@barbeau I defer to you defining when and how we could migrate the local web UI. Maybe that module could be preserved in this PR? Maybe the whole `web-app` and subfolders structure could be kept? @aababilov

Reply:
In the interest of getting the PR merged to master quickly, I'd say we merge this PR without it and then bring it back via a new PR. That way everyone can continue to iterate on the master branch.

Reply:
Fine with me. Just know the branch is broken because of it. Should be an easy fix @lionel-nj

Reply:
Just seeing this now! Thanks @fabrice-v for pointing out. I removed the jobs that relied on the existence of the local web ui()
I also added instructions to package the app with Gradle (maybe I should have just removed the workflow jobs that was triggered? I can still remove these lines of code if there is need). This led to also remove the `javadoc` related workflow jobs since they were also failing. I can always reintegrate them.